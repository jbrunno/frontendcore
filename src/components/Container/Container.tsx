import React, { ElementType } from 'react';
import {
  Container as MuiContainer,
  ContainerProps as MuiContainerProps,
} from '@mui/material';

export type ContainerProps = MuiContainerProps & {
  component?: ElementType;
};

export function Container(props: ContainerProps) {
  return <MuiContainer {...props} />;
}

export default Container;
