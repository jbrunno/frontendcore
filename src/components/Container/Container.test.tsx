import React from 'react';
import { render } from '@test-utils/component-testing';
import { Container } from './Container';

test('renders the Container', () => {
  const { container } = render(<Container />);
  expect(container).toMatchSnapshot();
});
