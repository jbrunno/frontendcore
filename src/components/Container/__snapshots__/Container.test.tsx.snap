// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the Container 1`] = `
.emotion-0 {
  width: 100%;
  margin-left: auto;
  box-sizing: border-box;
  margin-right: auto;
  display: block;
  padding-left: 1rem;
  padding-right: 1rem;
}

@media (min-width:600px) {
  .emotion-0 {
    padding-left: 1.5rem;
    padding-right: 1.5rem;
  }
}

@media (min-width:1200px) {
  .emotion-0 {
    max-width: 1200px;
  }
}

<div>
  <div
    class="MuiContainer-root MuiContainer-maxWidthLg emotion-0"
  />
</div>
`;
